charts/meilisearch/values.yaml

```diff
@@ -78,6 +78,15 @@ service:
 container:
   containerPort: 7700
 
+podSecurityContext: {}
+
+securityContext:
```

Comment:
Thanks for putting in this PR, as this eliminates something that I've had to manually patch. 

Might it also make sense to set default values for runAsUser, runAsGroup, runAsNonRoot and potentially also fsGroup (for persistence)? I've had all of those set without issue.

Reply:
@mike-stewart Could you please share - which values.yalm do you use with these settings? 
In my case I could not get meilisearch running with `runAsNonRoot: false` - it fails on the start and I see that the "official" image `getmeili/meilisearch:v1.2.0` still expects to be running with root user (ref: https://github.com/meilisearch/meilisearch/issues/2051, https://github.com/meilisearch/meilisearch/pull/2032).
But maybe I miss something 🤔

Reply:
@legal90 I have this, which seems to be working:
```yaml
securityContext:
  fsGroup: 2000
  runAsUser: 10000
  runAsGroup: 3000
  runAsNonRoot: true
```

A lot of the comments in that other issue seem to be about file ownership, which might be addressed by the fsgroup. I suspect you also need runAsUser set if runAsNonRoot is true and the dockerfile doesn't already specify a non-zero UID.

Reply:
Thank you, @mike-stewart! It seems I got it working now. 👍 
I think I will add these settings in a separate PR, with a minor version bump, since it can break some scenarios.

Reply:
Thanks @mike-stewart for your comment